### DISCUSSION TITLE
Welcome to NotSoSimpleConstruction Discussions!

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/NotSoSimpleConstruction#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Syndel-Markus** [comment]
There used to be a way to convert ore into metal then metal into rocket parts. I can't seem to find a way to do that anymore...

**Reply by zer0Kerbal** [reply]
@Syndel-Markus 
> There used to be a way to convert ore into metal then metal into rocket parts. I can't seem to find a way to do that anymore...

Do you have SimpleConstruction! (SCON) installed from CurseForge?

if so - then to convert from ore to metal use the stock ISRU's; to convert from metal to RocketParts use the stock Science Lab.
